tao_compiler/mlir/xla/ral/BUILD

```diff
@@ -327,7 +327,13 @@ cc_library(
         "-fopenmp",
         "-ldl",
         "-lm"
-    ],
+    ] + if_blade_gemm([
+        "-Wl,--start-group",
+        "$(location //tao/third_party/mkldnn:build)/intel/lib/libmkl_intel_ilp64.a",
```

Comment:
Can we use mkldnn download from bazel as `$(location @mkl_static//:lib)/libmkl_intel_ilp64.a` or so on.

Reply:
Done, and I've verified that it really works, as a result there's no need to download/build mkldnn in tao_build.py for gpu build. Thanks!